mkdocs_monorepo_plugin/parser.py

```diff
@@ -56,7 +56,8 @@ def removeMkdocsYmlFromPath(path):
             rootDir = os.path.normpath(os.path.join(
                 os.getcwd(), self.config['config_file_path'], '../'))
             absPath = os.path.normpath(os.path.join(
-                rootDir, re.sub(r'\/mkdocs.yml$', '', path, re.IGNORECASE)))
+                # Remove config file name from path, match any *.yml file
+                rootDir, re.sub(r'(.*)\/.*\.yml$', r'\1', path, re.IGNORECASE)))
```

Comment:
hmm what if someone uses a file name of "mkdocs.monorepo.yml", that will be accepted as well. Do you see any issues with accepting that? cc @OrkoHunter as well. 

Also I'm not sure this ex will match any .yml file

<img width="749" alt="Screen Shot 2021-02-10 at 11 01 42 AM" src="https://user-images.githubusercontent.com/25153114/107494669-5d047280-6b8f-11eb-84f3-df0e971f7d96.png">

instead if I do
<img width="720" alt="Screen Shot 2021-02-10 at 11 04 04 AM" src="https://user-images.githubusercontent.com/25153114/107494951-b79dce80-6b8f-11eb-8d21-b672ee25ff69.png">

I get matches.

Reply:
It's meant to match the file name including the .yml extension, so it can be stripped from the path. That's what `removeMkdocsYmlFromPath()` is for. You can play with the expression here: https://regex101.com/r/9CsHl2/1

Reply:
Understand, just wanted to highlight that it will not match any .yml file, but as its always going to be a path here I guess its fine in this case. But `(.*)\.yml` will cover both files and paths I assume.